packages/react-sdk/docusaurus/docs/React/02-guides/02-call-lifecycle.mdx

```diff
@@ -106,20 +143,23 @@ This is the happy path, when both parties join the active call. The
 sequenceDiagram
     participant Client_A
     participant Back-end
-    participant SFU
     participant Client_B
+    participant SFU
     Client_A->>Back-end: 1. Create a call
-    Back-end->>Client_B: 2. Back-end WS event 'call.created'
-    Client_B->>Back-end: 3. Accept the call
-    Back-end->>Client_A: 4. Back-end WS event 'call.accepted'
-    Client_B->>Back-end: 5. Ask for SFU to join the call
-    Client_A->>Back-end: 6. Ask for SFU to join the call
-    Client_B->>SFU: 7.  Join the call
-    Client_A->>SFU: 8. Join the call
-    SFU->>Client_B: 9. SFU WS event 'participant.joined'
-    Client_A->>SFU: 10. Leave the call
-    SFU->>Client_B: 11. SFU WS event 'participant.left'
-    Client_B->>SFU: 12. Leave the call
+    Back-end-->>Client_B: 2. Event: 'call.created'
+    Back-end-->>Client_B: 3. Event: 'call.ring'
+    Client_B->>Back-end: 4. Accept the call
+    Back-end-->>Client_A: 5. Event: 'call.accepted'
+    Client_B->>SFU: 6.  Join the call
+    Client_A->>SFU: 7. Join the call
+    Back-end-->>Client_A: 8. Event: 'call.session_started'
+    Back-end-->>Client_B: 9. Event: 'call.session_started'
+    Back-end-->>Client_B: 8. Event: 'call.session_participant_joined'
+    Client_A->>SFU: 9. Leave the call
+    Back-end-->>Client_B: 10. Event 'call.session_participant_left'
```

Comment:
Is this the coordinator or SFU?

Reply:
This is the coordinator. I don't think that we should be documenting the SFU events as it is in our best interest to keep them internal. Especially for this use case, where we have two different events (with slightly different data) for the very same thing.